spec/controllers/vm_cloud_controller_spec.rb

```diff
@@ -15,67 +15,53 @@
   # converted into proper routes and test is changed to test the new routes.
   describe 'x_button' do
     describe 'corresponding methods are called for allowed actions' do
-      ApplicationController::Explorer::X_BUTTON_ALLOWED_ACTIONS.each_pair do |action_name, method|
+      ApplicationController::Explorer::X_BUTTON_ALLOWED_ACTIONS.each do |action_name, method|
```

Comment:
I don't think we should change this.  It doesn't impact the behavior so it's just noise in the diff (also `each` in this case will allocate arrays where `each_pair` won't, but really the "diff noise" is my objection).